CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ FEATURES
 * Add support for pushing `consul-lambda-registrator` public image to private ecr repo through terraform.
   [[GH-82]](https://github.com/hashicorp/terraform-aws-consul-lambda/pull/82)
 
+* Add support for running `consul-lambda-registrator` in arm64 runtime and published arm64 artifacts for consul-lambda-registrator and consul-lambda-extension.
```

Comment:
Isn't there support for running `consul-lambda-extension` on the arm64 runtime too? Maybe this is better?

```suggestion
* Add  arm64 support to `consul-lambda-registrator` and `consul-lambda-extension`.
```